### PR TITLE
fix(mob): updating to align with new website structure

### LINKED ISSUE #972: Mob: request (new website design?)
https://www.mob.co.uk/recipes/roasted-tomato-spaghetti-soup
https://www.mob.co.uk/recipes/speedy-udon-miso-noodle-soup

### PR DESCRIPTION
**Pull request by jacksgreen**
Fixes #972

### CODE REVIEW

File: recipe_scrapers/mob.py [hunk 1]
```diff
@@ -1,55 +1,73 @@
...
+            self.soup.find("script", {"id": "__NEXT_DATA__"}).get_text()
+        )["props"]["pageProps"]["recipe"]
 
     @classmethod
     def host(cls, domain="mob.co.uk"):
         return domain
 
     def author(self):
-        return self.schema.author()
+        return "Mob"
```
[the remark this answers is not in the document]
Comment (jacksgreen) [reply recipe_scrapers/mob.py]: thanks, updated 👍

**Comment by jayaddison** [comment]
@jacksgreen could you update the description of this to include the text 'Fixes #977'?  (that'll mean that your bugreport will be automagically closed if+when this is merged).

**Comment by jacksgreen** [comment]
> Fixes #977

Sure, done 👍

**Comment by jayaddison** [comment]
Ah, oops: I think that was the PR title you updated, instead of the description field (the title is recommended to be fairly short, and the description is for longer-form information describing the pull request).

**Comment by jacksgreen** [comment]
> Ah, oops: I think that was the PR title you updated, instead of the description field (the title is recommended to be fairly short, and the description is for longer-form information describing the pull request).


Got you! Added now

**Comment by jacksgreen** [comment]
@jayaddison @CrossR I've updated the tests so they pass now, let me know if this can be merged!

**Review by CrossR — APPROVED** [review]

File: recipe_scrapers/mob.py [hunk 2]
```diff
@@ -72,10 +72,12 @@ def instructions(self):
         )
 
     def cuisine(self):
-        return self.recipe_json["cuisines"][0]["title"]
+        return " & ".join(
```
Comment (jayaddison) [inline recipe_scrapers/mob.py:75]: For consistency with a few other scrapers, let's use a comma-separated list for multiple categories here.  Anecdotally, that seems to be the way that many recipe sites list multi-categories too.

```suggestion
        return ", ".join(
```

Comment (jacksgreen) [reply recipe_scrapers/mob.py]: @jayaddison Sure, done!

Comment (jayaddison) [reply recipe_scrapers/mob.py]: Thanks again!

Comment (jayaddison) [reply recipe_scrapers/mob.py]: Ah, it looks like I commented on the wrong line here: this was the `cuisine` field, but I think we want these comma-separated lists for both `cuisine` and `category`.

(sorry, I should have noticed that during the earlier review)

Comment (jacksgreen) [reply recipe_scrapers/mob.py]: no problem, updated now 👍

**Comment by jayaddison** [comment]
Looks good, merging in a few moments - thanks @jacksgreen!  This will be included in an upcoming version of `recipe-scrapers` soon.

I'm going to fix one more detail in the description - I recommended linking this to #977, but that's this pull request (!) - instead what I should have written #972, the issue that this resolves/fixes.

**Pull request merged**